docs/api.md

```diff
@@ -25,30 +25,31 @@ For information about the Antrea API versioning policy, please refer to this
 These are the CRDs currently available in `crd.antrea.io`.
 
 | CRD | CRD version | Introduced in | Deprecated in / Planned Deprecation | Planned Removal |
-|---|---|---|---|---|
-| `AntreaAgentInfo` | v1beta1 | v1.0.0 | N/A | N/A |
-| `AntreaControllerInfo` | v1beta1 | v1.0.0 | N/A | N/A |
-| `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v2.0.0 |
-| `ClusterGroup` | v1alpha3 | v1.1.0 | v1.13.0 | N/A |
-| `ClusterGroup` | v1beta1 | v1.13.0 | N/A | N/A |
-| `ClusterNetworkPolicy` | v1alpha1 | v1.0.0 | v1.13.0 | N/A |
-| `ClusterNetworkPolicy` | v1beta1 | v1.13.0 | N/A | N/A |
-| `Egress` | v1alpha2 | v1.0.0 | N/A | N/A |
-| `Egress` | v1beta1 | v1.13.0 | N/A | N/A |
-| `ExternalEntity` | v1alpha2 | v1.0.0 | N/A | N/A |
-| `ExternalIPPool` | v1alpha2 | v1.2.0 | v1.13.0 | N/A |
-| `ExternalIPPool` | v1beta1 | v1.13.0 | N/A | N/A |
-| `ExternalNode`   | v1alpha1 | v1.8.0 | N/A | N/A |
-| `IPPool`| v1alpha2 | v1.4.0 | N/A | N/A |
-| `Group` | v1alpha3 | v1.8.0 | v1.13.0 | N/A |
-| `Group` | v1beta1 | v1.13.0 | N/A | N/A |
-| `NetworkPolicy` | v1alpha1 | v1.0.0 | v1.13.0 | N/A |
-| `NetworkPolicy` | v1beta1 | v1.13.0 | N/A | N/A |
-| `SupportBundleCollection` | v1alpha1 | v1.10.0 | N/A | N/A |
-| `Tier` | v1alpha1 | v1.0.0 | v1.13.0 | v2.0.0 |
-| `Tier` | v1beta1 | v1.13.0 | N/A | N/A |
-| `Traceflow` | v1alpha1 | v1.0.0 | v1.13.0 | N/A |
-| `Traceflow` | v1beta1 | v1.13.0 | N/A | N/A |
+|---|-------------|---------------|---|---|
```

Comment:
nit: it's better to avoid unnecessary formatting changes in this case, so we can focus on what was actually updated

Reply:
Got it, thanks.